app/controllers/application_controller.rb

```diff
@@ -36,6 +36,9 @@ def self.big_query_client
   around_action :set_current_user
   def set_current_user
     Current.user = current_user
+    if current_user.present?
+      current_user.update_api_last_access_at!
```

Comment:
since we're updating this on any calls, not just api calls, would a better name be just `update_last_access_at!` ?

Reply:
I thought about that, but the `last_access_at` is something stored in the Warden/Devise session client-side, and I didn't want there to be any confusion.  But I take the point - most people probably wouldn't note the distinction.

Reply:
5d2bcbc